packages/components/radio-group/src/radio-group.stories.ts

```diff
@@ -237,7 +237,9 @@ export const CustomValidation: StoryObj = {
         <sl-radio value="1">One</sl-radio>
         <sl-radio value="2">Two</sl-radio>
         <sl-radio value="3">Three</sl-radio>
-        <div slot="value-missing">This is the custom value-missing message (for the required attribute).</div>
+        <div slot="value-missing" error-size="sm">
```

Comment:
Should the error size be configurable? It feels like too much?

Reply:
Feels like the wrong approach for something that is pure styling. What about using a custom property?

```css
sl-input.mine {
  --sl-error-size: sm;
}
```

But you need container queries (style specifically) for that to work.

Perhaps
```
sl-input.mine {
  --sl-error-size: var(--sl-error-size-sm);
}
``` ?